### PR TITLE
added four web tests and related pages

### PR DESCRIPTION
**Pull request by ashchavinska**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    private ExtendedWebElement loginResult;
+
+    @FindBy(xpath = "//div[@class='main main-review l-box col float-right']//p")
+    private ExtendedWebElement loginFailReason;
+
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String isLoginSuccess() {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/LoginPage.java]: rename this method

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        Assert.assertTrue(header.isLogoPresent(), "Logo doesn't present");
+        Assert.assertTrue(header.isSearchFieldPresent(), "Search field doesn't present");
+        Assert.assertTrue(header.isTipIconPresent(), "Tip icon doesn't present");
+        Assert.assertTrue(header.isFacebookIconPresent(), "Facebook icon doesn't present");
+        Assert.assertTrue(header.isTwitterIconPresent(), "Twitter icon doesn't present");
+        Assert.assertTrue(header.isInstagramIconPresent(), "Instagram icon doesn't present");
+        Assert.assertTrue(header.isYouTubeIconPresent(), "YouTube icon doesn't present");
+        Assert.assertTrue(header.isRssIconPresent(), "Rss icon doesn't present");
+        Assert.assertTrue(header.isLoginIconPresent(), "Login icon doesn't present");
+        Assert.assertTrue(header.isSignUpIconPresent(), "SignUp icon doesn't present");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: use SoftAssert

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        Assert.assertTrue(homePage.isPageOpened(), "Home page doesn't open");
+
+        LoginField loginField = homePage.getHeader().openLoginField();
+        Assert.assertTrue(loginField.isLoginFieldPresent(), "Login field doesn't present");
+
+        loginField.enterEmail(invalid_email);
+        loginField.enterPassword(valid_password);
+        LoginPage loginPage = loginField.clickLoginButton();
+        Assert.assertEquals(loginPage.isLoginSuccess(), "Login failed.", "Login not failed");
+        Assert.assertEquals(loginPage.loginFailReason(), "Reason: User record not found.", "reason is different");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java:71]: use constant

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        Assert.assertTrue(homePage.isPageOpened(), "Home page doesn't open");
+
+        LoginField loginField = homePage.getHeader().openLoginField();
+        Assert.assertTrue(loginField.isLoginFieldPresent(), "Login field doesn't present");
+
+        loginField.enterEmail(valid_email);
+        loginField.enterPassword(invalid_password);
+        LoginPage loginPage = loginField.clickLoginButton();
+        Assert.assertEquals(loginPage.isLoginSuccess(), "Login failed.", "Login not failed");
+        Assert.assertEquals(loginPage.loginFailReason(), "Reason: Wrong password.", "reason is different");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java:88]: use constant

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 3 above)
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: use constants

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 4 above)
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: use constants

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/LoginPage.java (hunk 1 above)
Comment (ashchavinska) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/LoginPage.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 2 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**